sycl/doc/UsersManual.md

```diff
@@ -81,95 +134,84 @@ your application.
+      splitting device code. This is default mode.
 
-**`-fsycl-device-code-split`**
+**`-f[no-]sycl-device-lib=<lib1>[,<lib2>,...]`**
 
-    Perform SYCL device code split in the per_source mode, i.e. create a
-    separate device code module for each source (translation unit).
+    Enables/disables linking of the device libraries. Supported libraries:
+    libm-fp32, libm-fp64, libc, all. Use of 'all' will enable/disable all of
+    the device libraries.
 
-### Intel FPGA specific options
+## Intel FPGA specific options
 
 **`-fintelfpga`**
 
-    Perform ahead of time compilation for Intel FPGA, which relies on the
-    external tool `aoc` being available in the `PATH`.
-
-    This option is roughly equivalent to
-    `-fsycl-targets=spir64_fpga-unknown-unknown-sycldevice -g -MMD -lOpenCL`.
-
-    It is incompatible with `-fsycl-targets=...`; if ahead of time compilation
-    is needed for multiple backends (e.g. Intel FPGA, Intel GPU, etc.), the
-    alternative form based on
-    `-fsycl-targets=spir64_fpga-unknown-unknown-sycldevice` should be used
-    instead.
+    Perform ahead of time compilation for Intel FPGA. It sets the target to
+    FPGA and turns on the debug options that are needed to generate FPGA
+    reports. It is functionally equivalent shortcut to
+    `-fsycl-targets=spir64_fpga-unknown-unknown-sycldevice -g -MMD` on Linux
+    and `-fsycl-targets=spir64_fpga-unknown-unknown-sycldevice -Zi -MMD` on
+    Windows.
 
-**`-fsycl-link=<value>`**
+**`-fsycl-link=<output>`**
 
-    Generate partially linked device and host object to be used at various
-    stages of compilation. Takes the device binary(s) generated from a `-fsycl`
-    enabled compilation and wrap to create a host linkable object. This option
-    is enabled only in ahead of time compilation mode fore FPGA (i.e. when
-    `-fintelfpga` is set).
+    Controls FPGA target binary output format. Same as -fsycl-link, but
+    optional output can be one of the following:
+    * early - generate html reports and an intermediate object file that avoids
+    a full Quartus compile. Usually takes minutes to generate. Link can later
+    be resumed from this point using -fsycl-link=image.
+    * image - generate a bitstream which is ready to be linked and used on a
+    FPGA board. Usually takes hours to generate.
 
 **`-reuse-exe=<exe>`**
 
-    Speed up FPGA aoc compile if the device code in <exe> is unchanged.
+    Speed up FPGA backend compilation if the device code in <binary> is
+    unchanged. If it's safe to do so the compiler will re-use the device binary
+    embedded within it. This can be used to minimize or avoid long Quartus
+    compile times for FPGA targets when the device code is unchanged.
 
-### Other options
+## Other options
 
 **`-fsycl-device-only`**
 
-    Compile only SYCL device code.
+    Compile only device part of the code and discard host part.
 
-**`-fsycl-use-bitcode`**
+**`-fsycl-use-bitcode`** [DEPRECATED]
```

Comment:
I found this option quite useful for development/debug purposes. What is the suggested replacement? `-emit-llvm`?

Reply:
@mdtoguchi, is it a good replacement?

Reply:
`-fsycl-use-bitcode` is the default, so there is only `-fno-sycl-use-bitcode` that is useful when paired with `-fsycl-device-only`.  Is it possible for this to not be deprecated and stick around?  There is desire to provide support for .spv and .bc all the way to fat obj.

Reply:
Ok. Updated.